docs/ism/policies.md

```diff
@@ -8,13 +8,248 @@ has_children: false
+          "error_notification": null,
+          "default_state": "ingest",
+          "states": [
+              {
+                  "name": "ingest",
+                  "actions": [],
+                  "transitions": [
+                      {
+                          "state_name": "delete",
+                          "conditions": {
+                              "min_index_age": "30d"
+                          }
+                      }
+                  ]
+              },
+              {
+                  "name": "delete",
+                  "actions": [
+                      {
+                          "delete": {}
+                      }
+                  ],
+                  "transitions": []
+              }
+          ]
+      }
+  }
+  ```
+
+This policy puts the index in an “ingest” state by default. After a period of thirty days, this index is rolled over into a “delete” state where it’s permanently deleted.
+
+---
+
+## States  
+
+A state is the description of the status the managed index is currently in. A managed index can only be in one state at a time. Each state has associated actions that are executed sequentially on entering a state and transitions that are checked after all the actions have been completed.
+
+This table lists the parameters you can define for a state:
+
+Parameter | Description | Type | Required
+:--- | :--- |:--- |:--- |
+`name` |  The name of the state. | `string` | Yes
+`actions` | The actions to execute after entering a state. For more information, see [Actions](#actions). | `nested list of objects` | Yes
+`transitions` | The next states and conditions required to transition to those states, if no transitions are specified then the policy assumes that it's complete and can now stop managing the index. For more information, see [Transitions](#transitions). | `nested list of objects` | Yes
+
+---
+
+## Actions
+
+Actions define the steps that'll be sequentially executed on entering a specific state.
+
+This table lists the parameters you can define for an action:
+
+Parameter | Description | Type | Required | Default
+:--- | :--- |:--- |:--- |
+`timeout` |  The timeout period for the action, accepts time units for minutes, hours, and days. | `time unit` | No | specific to action
+`retry` | The retry configuration for the action. | `object` | No | specific to action
+`count` | The number of retry counts. | `number` | No | specific to action
+`backoff` | The backoff policy type to use when retrying. | `string` | No | exponential
+`delay` | The time to wait between retries, accepts time units for minutes, hours, and days. | `time unit` | No | specific to action
+
+All actions have common `timeout`, `retry`, and `backoff` settings. If they are not specified, appropriate defaults per action are used.
+
+Here's a sample action that has a timeout period of one hour, that is retried three times with the exponential backup policy, with a delay of five seconds between each retry:
```

Comment:
Passive voice. "timeout period of one hour. ISM retries three times with an exponential backup policy, with a delay of..."